policyengine_us/parameters/gov/states/ca/tax/income/alternative_minimum_tax/age_threshold/lower.yaml

```diff
@@ -0,0 +1,20 @@
+description: California limits the child exemption amount to filers with age below this lower threshold.
+values:
+  2021-01-01: 18
+
+metadata:
+  label: California alternative minimum tax child exemption lower age threshold 
+  period: year
+  unit: year
+  reference:
+    - title: 2021 Schedule P (540) Alternative Minimum Tax and Credit Limitations - Residents
+      href: https://www.ftb.ca.gov/forms/2021/2021-540-p.pdf#page=1
+    - title: 2022 Schedule P (540) Alternative Minimum Tax and Credit Limitations - Residents
+      href: https://www.ftb.ca.gov/forms/2022/2022-540-p.pdf#page=1
+    - title: 2022 Instructions for Schedule P (540) Line 22
+      href: https://www.ftb.ca.gov/forms/2022/2022-540-p-instructions.html
+    - title: 2021 Instructions for Schedule P (540) Line 22
+      href: https://www.ftb.ca.gov/forms/2021/2021-540-p-instructions.html
+    # The legal code does not specify the age threshold
```

Comment:
i think this is based on the internal revenue code: https://chat.openai.com/share/46cfecf8-fe9f-4520-8d59-a2a9f2c819a2

it matches the definition of children in [IRC 152(c)(3)](https://www.law.cornell.edu/uscode/text/26/152#c_3)